geospace-http/src/main/scala/com/socrata/geospace/http/client/CoreServerClient.scala

```diff
@@ -117,7 +117,7 @@ class CoreServerClient(httpClient: HttpClient,
       case Some(rb) =>
         val request = buildRequest(rb)
         logger.info("Request: " + request)
-        for (response <- httpClient.execute(request)) yield {
+        for (response <- httpClient.execute(request)) {
```

Comment:
Let me investigate, i believe it does work, something about managed not being expected any longer.